src/components/StyledModal.js

```diff
@@ -24,14 +24,20 @@ const ModalWrapper = styled(Container)`
     `}
 `;
 
+const GlobalModalStyle = createGlobalStyle`
+  body {
+    overflow: ${props => (props.show ? 'hidden' : 'auto')};
```

Comment:
It's better, but it will still override the page `overflow`. Remember that for example `Body.js` sets `overflow-x` to `hidden`.

If the modal is hidden, then we should not modify the `overflow`.

Reply:
So I should use `overflow-y` instead?

Reply:
Either 

```css
const GlobalModalStyle = createGlobalStyle`
  body {
   ${props => props.show && css`
     overflow: hidden;
   `}
  }
`
```

Or in the render func

```es6
...
{show && (
  <GlobalModalStyle />
)}
```

I personally prefer the (2), but both are valid.

Reply:
The general idea is: we shouldn't touch the global styles when the modal is not visible.